### ISSUE TITLE
Ztunnel log improvements

### ISSUE THREAD

**Issue by howardjohn** [istio/istio#44541]
```
$ ik pc log ds/ztunnel -n istio-system --level=ztunnel::pool=debug
Error: unrecognized logging level: :pool=debug
```

We are validating this incorrectly

Also isZtunnelPod is creating many new clients, we should just create one and use it

**Comment by jaellio** [comment]
Is `ztunnel::pool=debug` following the log level form? Is `ztunnel::pool` the logger name?

**Comment by howardjohn** [comment]
Yes, in the rust logging the following are valid

debug
foo=debug
foo::bar::baz=debug

:: are basically namespacing it. Unlike envoy we don't really have a statically known set of logging scopes

**Comment by jaellio** [comment]
> Yes, in the rust logging the following are valid
> 
> debug foo=debug foo::bar::baz=debug
> 
> :: are basically namespacing it. Unlike envoy we don't really have a statically known set of logging scopes

Got it! Thanks for clarifying!

**Comment by keithmattix** [comment]
@MorrisLaw is looking to take this

**Comment by keithmattix** [comment]
Fixed by #45022
